common/dds-model.cpp

```diff
@@ -104,6 +104,9 @@ rs2::dds_model::priority rs2::dds_model::classifyPriority( link_priority & pr )
 
 bool dds_model::check_DDS_support()
 {
+    if (!_device.supports(RS2_CAMERA_INFO_PRODUCT_LINE) || !(std::string(_device.get_info(RS2_CAMERA_INFO_PRODUCT_LINE)) == "D500"))
```

Comment:
1. lets please check specifically D555 Camera name
2. Maybe the readability with be more clear with something like:
```
if !(supports && D555)
    return false
```